test/Lombiq.OSOCE.Tests.UI/Tests/ThemeTests/BlogBaseThemeTests.cs

```diff
@@ -45,6 +46,50 @@ public BlogBehaviorBaseThemeTests(ITestOutputHelper testOutputHelper)
             },
             browser);
 
+    [Theory, Chrome]
+    public Task ContentMenuItemShouldWorkCorrectly(Browser browser) =>
+        ExecuteTestAfterSetupAndThemeSwitchAsync(
+            async context =>
+            {
+                await context.SignInDirectlyAsync();
+                await context.GoToRelativeUrlAsync("/Admin/Contents/ContentItems/Menu");
```

Comment:
Also below, and do something similar for the form action (see `context.AdminUrlPrefix` or `context.GetAbsoluteAdminUri()`.
```suggestion
                await context.GoToAdminRelativeUrlAsync("/Contents/ContentItems/Menu");
```

Reply:
> do something similar for the form action

I can just make the query partial, like `form[action*='SetCurrentTheme/Lombiq.BaseTheme.Samples'] button` because only the theme name matters there.